widgetssdk/build.gradle

```diff
@@ -80,6 +80,11 @@ tasks.register('publishToLocalMaven') {
   dependsOn ':widgetssdk:publishSnapshotPublicationToMavenLocal'
 }
 
+tasks.withType(Test).configureEach {
+  // Increase the maximum heap size to 2 GB
```

Comment:
Please comment 'why' this configuration is needed.

Reply:
This was needed with Paparazzi 1.3.4 to not fail while running on the local machine; I will recheck for the 1.3.3 version.